galaxykit/containers.py

```diff
@@ -24,15 +24,17 @@ def delete_container(client, name):
     """
     Delete container
     """
-    delete_url = f"v3/plugin/execution-environments/repositories/{name}/"
+    delete_url = (
+        f"{client.ui_ee_endpoint_prefix}execution-environments/repositories/{name}/"
+    )
     return client.delete(delete_url, parse_json=False)
 
 
 def create_container(client, name, upstream_name, registry):
     """
     Create container
     """
-    create_url = f"_ui/v1/execution-environments/remotes/"
+    create_url = f"{client.ui_ee_endpoint_prefix}execution-environments/remotes/"
```

Comment:
this endpoint doesn't exist ..
looks like you may have fixed https://github.com/ansible/galaxykit/pull/80#discussion_r1020171221 and then undone the change somehow?